index.html

```diff
@@ -0,0 +1,188 @@
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="#sponsors">Sponsor</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-style" href="#featureDancer">News</a>
+            </li>
+            <li class="header-botoom-list">
+              <a class="nav-bottom-links-campaing" href="#">Join</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="section-header">
+      <div class="section-header-content">
+        <h1 class="section-header-title">Hello! Dance with me, world!</h1>
+        <h2 class="section-header-second-title">
+          BALCAN SALSA CONGRESS <br />
+          SUMMER <br />
+          <span class="section-header-second-title-year">2021</span>
+        </h2>
+        <p class="section-header-text-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus
+          iusto, maxime nulla, nesciunt reprehenderit!
+        </p>
+        <h2 class="section-header-third-title">2021.08.06(FRI) ~ 08(SUN)</h2>
+        <p class="section-header-info">
+          @ National Palace of Culture, Art Center Sofia and more
+        </p>
+      </div>
+    </section>
+    <main id="main">
+      <section class="main-program" id="program">
+        <h3 class="main-program-title text-center">Main Program</h3>
+        <p class="footer-top-underline display-flex-center">
+          <span class="footer-top-underline-span"></span>
+        </p>
+        <ul class="main-program-cards">
+          <li class="main-program-cards-content">
+            <i class="fas fa-trophy"></i>
+            <p class="main-program-cards-title">Lecture</p>
+            <span class="main-program-cards-text"
+              >Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              aliquid.</span
+            >
+          </li>
+          <li class="main-program-cards-content">
+            <i class="fas fa-trophy"></i>
+            <p class="main-program-cards-title">Lecture</p>
+            <span class="main-program-cards-text"
+              >Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              aliquid.</span
+            >
+          </li>
+          <li class="main-program-cards-content">
+            <i class="fas fa-trophy"></i>
+            <p class="main-program-cards-title">Lecture</p>
+            <span class="main-program-cards-text"
+              >Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              aliquid.</span
+            >
+          </li>
+          <li class="main-program-cards-content">
+            <i class="fas fa-trophy"></i>
+            <p class="main-program-cards-title">Lecture</p>
+            <span class="main-program-cards-text"
+              >Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum,
+              aliquid.</span
+            >
```

Comment:
[OPTIONAL]

- Instead of using same content (`Lorem ipsum ...`) repeatedly, you can use customized content to make your website realistic.